cms/envs/common.py

```diff
@@ -190,6 +190,9 @@
     'ENABLE_SPECIAL_EXAMS': False,
 
     'ORGANIZATIONS_APP': False,
+
+    # Show LMS Language selector
```

Comment:
This is not an LMS feature flag -- also the comment is somewhat redundant -- I would either change it to provide more context or remove it.